terraform/modules/networking/nacl-private.tf

```diff
@@ -29,6 +29,30 @@ resource "aws_network_acl_rule" "private_ingress_ephemeral_vpc" {
   to_port        = 65535
 }
 
+resource "aws_network_acl_rule" "private_ingress_https_vpc" {
```

Comment:
Do we need one rule for 80 now?

Reply:
Hi Raghu , we have a separate rule for port  80 . 

resource "aws_network_acl_rule" "private_egress_http_vpc" {
  network_acl_id = aws_network_acl.private.id
  rule_number    = 300
  egress         = true
  protocol       = "tcp"
  rule_action    = "allow"
  cidr_block     = "10.0.0.0/16"
  from_port      = 80
  to_port        = 80
}